src/clojure_tictactoe/game/game_play.clj

```diff
@@ -8,27 +8,35 @@
             [clojure-tictactoe.game.game-rules :as rules]
             ))
 
-(defn player-move [move-function board]
-  (board/mark-space (move-function board) @players/player-atom board))
+(defn player-move [move-function board marker]
+  (board/mark-space (move-function board) marker board))
 
-(defn continue-game [board]
-  (let [new-board (player-move (players/choose-player-function @players/player-atom) board)]
+(defn continue-game [board game-type markers turn]
+  (let [new-board (player-move
+                    (players/choose-player-function
+                      game-type turn)
+                    board
+                    (markers
+                      (players/current-player turn)))]
```

Comment:
The things happening within this let variable are a little crazy and difficult to read. Can you clean this up a bit?

Reply:
✅